02-starting-with-data.Rmd

```diff
@@ -136,8 +137,27 @@ To open the dataset in RStudio's Data Viewer, use the `view()` function:
 view(surveys)
 ```
 
-> ### Note
->
+## Missing values in data
+Missing values are those not stored for a variable in the observation of interest. 
+This is a common problem across all domains of research. The reasons could be many. 
+Missing values may have a significant effect on the conclusions of the research. 
+When viewing above data, we observe a lot of cells showing ‘NA’ values. 
+Those are missing values. While care should be taken to collect data so that there 
+are no missing values, most of the times values go missing. 
```

Comment:
I'd argue that values rarely "go missing". Usually the data can't be collected in the first place.